src/data/CCGraph.mli

```diff
@@ -35,17 +34,12 @@ type 'a iter_once = 'a iter
 exception Iter_once
 (** Raised when a sequence meant to be used once is used several times. *)
 
-module Iter : sig
-  type 'a t = 'a iter
+module Iter : sig type 'a t = 'a iter
 
-  val return : 'a -> 'a t
-  val ( >>= ) : 'a t -> ('a -> 'b t) -> 'b t
-  val map : ('a -> 'b) -> 'a t -> 'b t
-  val filter_map : ('a -> 'b option) -> 'a t -> 'b t
-  val iter : ('a -> unit) -> 'a t -> unit
-  val fold : ('b -> 'a -> 'b) -> 'b -> 'a t -> 'b
-  val to_list : 'a t -> 'a list
-end
+  val return : 'a -> 'a t val ( >>= ) : 'a t -> ('a -> 'b t) -> 'b t val map :
+  ('a -> 'b) -> 'a t -> 'b t val filter_map : ('a -> 'b option) -> 'a t -> 'b t
+  val iter : ('a -> unit) -> 'a t -> unit val fold : ('b -> 'a -> 'b) -> 'b ->
+  'a t -> 'b val to_list : 'a t -> 'a list end
```

Comment:
what happened here? :)

Reply:
I think it is ocamlformat. I will correct this and force push.